src/block.rs

```diff
@@ -47,23 +93,81 @@ pub struct BlockHeader {
     /// Block height
     pub height: u32,
     /// Block height
```

Comment:
You seem to have copied this comment from above..

Reply:
Lol, I noticed that and thought to fix it ... but apparently did not.